src/server/apps/worldserver/worldserver.conf.dist

```diff
@@ -2744,7 +2744,6 @@ Quests.IgnoreAutoComplete = 0
 #    Rate.RewardBonusMoney
 #        Description: Allows to further tweak the amount of extra money rewarded by quests when the player
 #                     is at MaxPlayerLevel (this amount is specified in quest_template.RewardBonusMoney).
-#                     NOTE: the final amount will also affected by Rate.Drop.Money
```

Comment:
Also, it makes no sense that a rate intended for how much money is dropped from creatures should affect the money from quests.

They are two different rates that should be calculated and handled separately for their intended purposes